tests/cupyx_tests/scipy_tests/signal_tests/test_fir_filter_design.py

```diff
@@ -325,8 +325,7 @@ def test_firls(self, xp, scp):
         N = 11  # number of taps in the filter
         a = 0.1  # width of the transition band
         # design a halfband symmetric low-pass filter
-        h = scp.signal.firls(N, [0, a, 0.5-a, 0.5], [1, 1, 0, 0], fs=1.0)
-        return h
+        return scp.signal.firls(N, [0, a, 0.5-a, 0.5], [1, 1, 0, 0], fs=1.0)
```

Comment:
Q: why do we change it here?

Reply:
Probably this is a left over of when I was fixing the tests

Reply:
ah I see, will fix it to minimize diff, thanks Emilio

Reply:
fixed in [7af49ef](https://github.com/cupy/cupy/pull/8442/commits/7af49ef482d76913bc0e4ca14d386d1e75657a06)